docs/pages/push-notifications/overview.mdx

```diff
@@ -4,36 +4,52 @@ sidebar_title: Overview
 description: An overview of Expo's push notification service.
 ---
 
+import { AnnotationQuestionIcon } from '@expo/styleguide-icons/outline/AnnotationQuestionIcon';
 import { Bell03Icon } from '@expo/styleguide-icons/outline/Bell03Icon';
+import { CheckDone02Icon } from '@expo/styleguide-icons/outline/CheckDone02Icon';
+import { Send03Icon } from '@expo/styleguide-icons/outline/Send03Icon';
+import { TerminalIcon } from '@expo/styleguide-icons/outline/TerminalIcon';
+
+Expo simplifies implementing push notifications by handling much of the complexity involved in communicating with Firebase Cloud Messaging (FCM) or Apple Push Notification Service (APNs). This allows you to treat Android and iOS notifications in the same way and save time both on the front-end and back-end.
+
+Follow the video or links below to learn how to set up push notifications, send them, and handle incoming notifications in your app.
 
 import { BoxLink } from '~/ui/components/BoxLink';
+import { NotificationsVideo } from '~/ui/components/NotificationsVideo';
 
-Expo makes implementing push notifications easy. All the hassle with device information and communicating with Firebase Cloud Messaging (FCM) or Apple Push Notification Service (APNs) is done behind the scenes. This allows you to treat Android and iOS notifications in the same way and save time both on the front-end and back-end.
+<NotificationsVideo />
+
+<BoxLink
+  title="What you need to know about notifications"
+  description="Different kinds of notifications and notification behaviors you need to know before you get started."
+  href="/push-notifications/what-you-need-to-know"
+  Icon={CheckDone02Icon}
```

Comment:
The page looks odd if all icons are Bell icons. I'm not sure how strong the association with what is on expo.dev is. 🤔

Reply:
For services associated with the expo.dev, we use the same icon as a visual navigator to represent that these docs belong to the same group, whether the BoxLinks to that particular section is linked within this section or from another section.

This visual indicator does help (would be a small % of users) to connect the dots and also helps us to maintain this consistency in form of a pattern. Good examples are any EAS docs (Build, Update, etc.). 

In the past, when we were not using icons that referenced the service icon, it became a little hard for us to maintain because each contributor was left with the decision of what icon to put in the box link.

Another way this helps us to indicate using other icon patterns such as when we use a generic docs icon(just a standalone guide)/learn section (referencing to the tutorial section)/SDK reference; it helps connect the dots that way as well.

Reply:
updated to `Bell03Icon` 👍